libraries/CoreAudio/CoreAudio.cpp

```diff
@@ -130,7 +135,9 @@ void CoreAudio::action( int id ) {
     case ENT_CLASH:
       if (soundPlayFlashRaw.isPlaying())
       {
-        soundPlayFlashRaw.stop();
+        //soundPlayFlashRaw.stop();
```

Comment:
please remove the commented line. It's fine to have the previous version in the history and a cleaner code